tendrl/node_agent/message/handler.py

```diff
@@ -35,6 +36,12 @@ def read_socket(self, sock, *args):
             frmt = "=%ds" % size
             msg = struct.unpack(frmt, data)
             message = Message.from_json(msg[0])
+            # Logger is in commons so passing alert from here
+            if message.priority == ALERT_PRIORITY:
+                update_alert(
```

Comment:
define "update_alert" for tendrl.integrations.gluster.alert and call that function too from here.

eg:  node_agent.alert.update_alert
       integrations.gluster.alert.update_alert

Reply:
@r0h4n why we need one more update_alert function in gluster-integration. Except for volume alert count calculation, there is no integration related operation are happening in node-agent update_alert function.


update_alert function just read /alerting/nodes or /alert/clusters directory in etcd and compare the alert with new incoming alert and update  in etcd. 

other than this it will find alert count of host level alert count and cluster level alert count. and volume level alert count. 
Here only volume alert count is integration specific. other than this nothing is integration specific.

Reply:
for this change, same logic repeats twice